src/Skia/Avalonia.Skia/DrawingContextImpl.cs

```diff
@@ -653,6 +671,36 @@ private void ConfigureVisualBrush(ref PaintWrapper paintWrapper, IVisualBrush vi
             }
         }
 
+        static SKColor SimpleColorBurn(SKColor bg, SKColor fg)
```

Comment:
We need to calculate that color properly, that code was here only for example purposes

Reply:
Removed, as we cant, you need a blending function that can work with the background bitmap. we dont have access to this.

Maybe for in - app acrylic in future.